src/electron/platform/android/scan-results.ts

```diff
@@ -67,4 +69,12 @@ export class ScanResults {
             return 'no-version';
         }
     }
+
+    public get screenshotData(): ScreenshotData {
```

Comment:
Can you please add these 2 test cases to scan-results-test.ts:
`screenshotData is null if missing from input`
`screenshotData is correct if specified in input`

You'll also need to update `buildScanResultsObject` in scan-results-helpers.ts. Feel free to ping me if the pattern used there seems unfamiliar.

Reply:
addressed and fixed in the 4th commit